example/burrow_integration/integration_eaxample.py

```diff
@@ -0,0 +1,171 @@
+    no_of_param=2
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding('coin#domain') #asset id
+    params=params+argument_encoding('500') # amount of asset
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, callee=address, input=params)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response=net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+@trace
+def create_account(address) :
+    params = get_first_four_bytes_of_keccak(b'createAccount(string,string,string)') 
+    no_of_param=3
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding('test') #source account id
+    params=params+argument_encoding('domain') #domain id
+    params=params+argument_encoding(test_private_key) #  key
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, callee=address, input=params)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response=net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+
+@trace
+def transfer(address) :
+    params = get_first_four_bytes_of_keccak(b'transferAsset(string,string,string,string)') 
+    no_of_param=4
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding(ADMIN_ACCOUNT_ID) #source account
+    params=params+argument_encoding('userone@domain') #destination account
+    params=params+argument_encoding('coin#domain') #asset id
+    params=params+argument_encoding('100') # amount of asset
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, callee=address, input=params)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response=net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+
+@trace
+def balance(address) :
+    params = get_first_four_bytes_of_keccak(b'queryBalance(string,string)')
+    no_of_param=2
+    for x in range(no_of_param) :
+        params=params+left_padded_address_of_param(x,no_of_param)
+    params=params+argument_encoding(ADMIN_ACCOUNT_ID) #account id
+    params=params+argument_encoding('coin#domain') #asset id
+    tx = iroha.transaction([
+        iroha.command('CallEngine', caller=ADMIN_ACCOUNT_ID, callee=address, input=params)
+    ])
+    IrohaCrypto.sign_transaction(tx, ADMIN_PRIVATE_KEY)
+    response=net.send_tx(tx)
+    for status in net.tx_status_stream(tx):
+        print(status)
+    hex_hash = binascii.hexlify(IrohaCrypto.hash(tx))
+    return hex_hash
+    
+    
+    
+hash = create_contract()
+address = get_engine_receipts(hash)
+hash = balance(address)
+get_engine_receipts(hash)
```

Comment:
I suggest some way of extracting return value from the function into human readable form. When You are extracting balance it would be better to print its directly from python code, without external online tools